packages/jest-circus/src/utils.ts

```diff
@@ -159,14 +159,15 @@ function checkIsError(error: unknown): error is Error {
 }
 
 export const callAsyncCircusFn = (
-  fn: Circus.AsyncFn,
+  testOrHook: Circus.TestEntry | Circus.Hook,
   testContext: Circus.TestContext | undefined,
-  asyncError: Circus.Exception,
   {isHook, timeout}: {isHook?: boolean | null; timeout: number},
```

Comment:
```suggestion
  {isHook, timeout}: {isHook: boolean; timeout: number},
```

since we're in the neighborhood 🙂 Dunno if it causes type errors, but if so they should be fixed